doc/examples/filters/plot_deconvolution.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""
+=====================
+Image Deconvolution
+=====================
+In this example, we deconvolve an image using Richardson-Lucy
+deconvolution algorithm.
+Richardson-Lucy deconvolution
+-----------------------------
```

Comment:
I would say, remove lines 8 and 9 here — an intermediate heading doesn't add much to this small example.